docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,92 @@
+# Allowing Groups to be Based on and Synced With External LDAP Records
+
+Linking internal `Group`s of users to their representations on LDAP servers allows for administrators to have an easier way to manage groups inside OpenShift. A command like `oadm sync groups` or `oadm sync group <name>` will trigger a sync of group members with their external support (if any).
```

Comment:
Describe exactly what we're going to sync.  I think that you're only going to be sync-ing users (both additions and deletions) in a group.